common.h2m

```diff
@@ -0,0 +1,16 @@
+[see-also]
```

Comment:
I deleted the files in top level directory and ran the script in `/man/man1`, however all files are regenerated anyway because the script tries to generate man file for all files and not just a specific file